packages/components/src/components/Snackbar/SnackbarProvider.tsx

```diff
@@ -0,0 +1,33 @@
+import React, { ReactNode, createContext, useState } from 'react'
+
+import { SNACKBAR_DEFAULT_OFFSET, Snackbar } from './Snackbar'
+import { ToastProvider } from 'react-native-toast-notifications'
+
+type SnackbarContextType = {
+  offset: number
+  setOffset: (newOffset: number) => void
+}
+
+export const SnackbarContext = createContext<SnackbarContextType | undefined>(
```

Comment:
Not sure if there's special guidance around the create-/use-Context, but should this be named like a global const/variable? AKA should it be all caps w/ underscores to ensure no conflicts with what consumers may be doing?

Reply:
Nvm, see below it's used as a component so probably good the way it is.